### PR TITLE
[12.0][REF] Convert edoc_purpose selection in fiscal constant

### PR DESCRIPTION
**Pull request by renatonlima**

### CODE REVIEW

**Review by marcelsavegnago — APPROVED** [review]

**Review by antoniospneto — APPROVED** [review]

**Review by mbcosta — APPROVED** [review]

**Review by rvalyi — APPROVED** [review]

**Comment by rvalyi** [comment]
mudança trivial -> fast tracking
/ocabot merge patch

**Comment by rvalyi** [comment]
dei um cancel no merge para não dar BO com esse merge que tava rolando ainda https://github.com/OCA/l10n-brazil/pull/1923 
vemos depois.

**Comment by rvalyi** [comment]
vish... O serie enrolada pro Travis rodar o merge essa v12... Agora vai...
/ocabot merge patch

**Comment by rvalyi** [comment]
> It looks like something changed on `12.0` in the meantime.
> Let me try again (no action is required from you).
> Prepared branch [12.0-ocabot-merge-pr-1933-by-rvalyi-bump-patch](https://github.com/OCA/l10n-brazil/commits/12.0-ocabot-merge-pr-1933-by-rvalyi-bump-patch), awaiting test results.

https://user-images.githubusercontent.com/16926/170800048-27517be8-3f83-491b-881a-96ff9c732277.mp4
